src/world_renderer.rs

```diff
@@ -280,6 +324,26 @@ impl WorldRenderer {
         Texture2d::new(display, image).unwrap()
     }
 
+    /// Loads a texture,cut it and returns it
+    fn load_texture_cut(bytes: &[u8], display: &Display<WindowSurface>, cut: Vec<[u32; 4]>) -> Texture2dArray {
```

Comment:
This function needs more documentation. It's not clear how _cut_ encodes the why, or why we need to cut an image. Just a little sentence that explains why we are cutting textures is enough here